examples/mining_ar.py

```diff
@@ -0,0 +1,132 @@
+          'association rules found in this small dataset. This happens because we did not '
+          f'set up the so called {COLOR_CODES["green"]}support{COLOR_CODES["default"]} '
+          'value. This is an indication of how frequently the itemset appears '
+          'in the dataset, expressed as the ratio of the number of transactions '
+          'containing A and B to the total number of transactions. '
+          'Since the default support value is 0, the system discovers '
+          'all association rules, even those that are present only '
+          "once in the dataset. Now, let's see the results with "
+          f'{COLOR_CODES["yellow"]}minsup=0.5{COLOR_CODES["default"]} and '
+          f'{COLOR_CODES["yellow"]}minconf=0.5{COLOR_CODES["default"]}.\n')
+    algo.execute(minsup=0.5, minconf=0.5)
+    print_ars(algo.get_ars())
+    print('\nNow you can see that the number of association rules have decreased '
+          'significantly. This happened due to minsup being set to 0.5. '
+          'Unfortunately, if you want to know what the support value is for a '
+          "particular association rule in a dataset, you can't get it with Desbordante.\n"
+          '\nA typical approach to controlling the algorithm is to employ '
+          f'{COLOR_CODES["bold_yellow"]}"usefulness"{COLOR_CODES["default"]}, '
+          'which is defined as confidence * support. In the last example, we set up min '
+          '"usefulness" = 0.5 * 0.5 = 0.25. \n\nNow, let\'s try with '
+          f'{COLOR_CODES["green"]}minsup=0.7{COLOR_CODES["default"]}, '
+          f'{COLOR_CODES["green"]}minconf=0.7{COLOR_CODES["default"]} '
+          f'and {COLOR_CODES["bold_yellow"]}"usefulness"=0.49{COLOR_CODES["default"]}\n')
+    algo.execute(minsup=0.7, minconf=0.7)
+    print_ars(algo.get_ars())
+    print('\nSo, now the total number of returned association rules is only four. '
+          'We reduced the amount of "noisy" information in our output. '
+          'You are free to play with these parameters to see how it changes things. '
+          'Eventually, you will find out what fits your dataset and your task best.')
+
+
+def scenario_singular():
+    algo = desbordante.ar.algorithms.Default()
+    algo.load_data(table=(TABLE_SINGULAR, ',', True), input_format='tabular')
+    algo.execute()
+    table = pandas.read_csv(TABLE_SINGULAR, header=None, index_col=0)
+
+    print("\nFor the second example, let's take a dataset with the same receipts "
+          'from the same supermarket, changing the input format to input_format='
+          '"singular". This is a two-column format, where the first column is the order '
+          'of the items, and the second column is the item that belongs to that order.\n')
+    print_table(table)
+    print('\nThis format is just a different table representation. Desbordante '
+          'is perfectly capable of the discovered rules from such tables, ' 
```

Comment:
"is perfectly capable of rules discovery in such tables, and the discovered objects are represented by the same type of objects as in the previous examples"